libs/ui/select/brain/src/lib/brn-select.service.ts

```diff
@@ -55,14 +56,14 @@ export class BrnSelectService {
 	constructor() {
 		this.listBoxValueChangeEvent$.pipe(takeUntilDestroyed()).subscribe((listBoxChange) => {
 			const updatedSelections = this.multiple() ? this.getUpdatedOptions(listBoxChange) : [listBoxChange.option];
+			const value = this.multiple() ? listBoxChange.value : listBoxChange.value[0];
 			this.state.update((state) => ({
 				...state,
 				selectedOptions: [...updatedSelections],
-				value: listBoxChange.value as string[],
+				value: value as string | string[],
 			}));
 		});
-
-		this.multiple$.pipe(takeUntilDestroyed()).subscribe((multiple) => {
+		this.multiple$.pipe(takeUntilDestroyed(), pairwise()).subscribe(([, multiple]) => {
```

Comment:
Why is pairwise needed here?

Reply:
also `takeUntil` should generally be placed always at the end of the operator chain

Reply:
I kind of abuse it here, as toObservable always emits the first value of the signal. E.g. if we set multiple is default false, this subscription will trigger with false. The forms API on the other hand is setting value before this is triggered, so we would deselect the preset value there. With pairwise the subscription only gets triggered if we have a previous value, so we only trigger this code if the multiple property was actively changed which we try to catch there by deselecting all items. If there is a better operator i would kindly use it.

Reply:
Maybe skip(1) will work too? 
Or have I missunderstood what your trying to achieve?